www/src/components/graphiql-embed.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+const GraphiQLEmbed = ({ title, url, query }) => {
+  const encodedQuery = encodeURIComponent(query)
+  return (
+    <iframe
+      title={title}
+      src={`${url}?query=${encodedQuery}&explorerIsOpen=false`}
+      width="600"
+      height="400"
+      lazy="true"
```

Comment:
```suggestion
      loading="lazy"
```

We currently use that and refers to: https://web.dev/native-lazy-loading/